samples/client/petstore/php/nullable-object-property-php/lib/Model/ModelWithNullableObjectProperty.php

```diff
@@ -0,0 +1,410 @@
+<?php
+/**
+ * ModelWithNullableObjectProperty
+ *
+ * PHP version 7.3
+ *
+ * @category Class
+ * @package  OpenAPI\Client
+ * @author   OpenAPI Generator team
+ * @link     https://openapi-generator.tech
+ */
+
+/**
+ * Title
+ *
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: latest
+ * Generated by: https://openapi-generator.tech
+ * OpenAPI Generator version: 5.3.1-SNAPSHOT
+ */
+
+/**
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+namespace OpenAPI\Client\Model;
+
+use \ArrayAccess;
+use \OpenAPI\Client\ObjectSerializer;
+
+/**
+ * ModelWithNullableObjectProperty Class Doc Comment
+ *
+ * @category Class
+ * @package  OpenAPI\Client
+ * @author   OpenAPI Generator team
+ * @link     https://openapi-generator.tech
+ * @implements \ArrayAccess<TKey, TValue>
+ * @template TKey int|null
+ * @template TValue mixed|null
+ */
+class ModelWithNullableObjectProperty implements ModelInterface, ArrayAccess, \JsonSerializable
+{
+    public const DISCRIMINATOR = null;
+
+    /**
+      * The original name of the model.
+      *
+      * @var string
+      */
+    protected static $openAPIModelName = 'ModelWithNullableObjectProperty';
+
+    /**
+      * Array of property to type mappings. Used for (de)serialization
+      *
+      * @var string[]
+      */
+    protected static $openAPITypes = [
+        'property_name30' => 'PropertyType',
```

Comment:
We just noticed that we still have a problem for PHP.

This name must be typed with namespaces for ObjectSerializer to be able to handle it